hibernate-reactive-core/src/test/java/org/hibernate/reactive/WrongCredentialsTest.java

```diff
@@ -28,6 +31,9 @@
  */
 public class WrongCredentialsTest extends BaseReactiveTest {
 
+	@Rule
+	public DatabaseSelectionRule dbRule = DatabaseSelectionRule.skipTestsFor( H2 );
```

Comment:
Instead of skipping the test, I would prefer if we'd keep H2 as a special case in the test (I'm assuming H2 should never fail the credentials check because it doesnt's have one)

Reply:
Other than wrapping the assertion in `if( dbType() != H2 ) {...}`, I'm not sure what else to do. The test doesn't persist anything, so the credentials should be checked during connection.

Reply:
No, you are right. It's fine like this